WcaOnRails/app/javascript/edit-events/ButtonActivatedModal.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import cn from 'classnames'
+import Modal from 'react-bootstrap/lib/Modal'
+import Button from 'react-bootstrap/lib/Button'
+import addEventListener from 'react-overlays/lib/utils/addEventListener';
+import ownerDocument from 'react-overlays/lib/utils/ownerDocument';
+
+export default class extends React.Component {
+  constructor() {
+    super();
+    this.state = { showModal: false };
+  }
+
+  open = () => {
```

Comment:
Ah! Because I wrote that code in `EditEvents` class before I knew you could define instance methods using the arrow function. I've changed the `EditEvents` class to get rid of the binds.

Reply:
Makes sense, to be honest I was surprised seeing this property assignment style, I was sure it's not a part of the ES6 spec, but we have a babel [plugin](https://github.com/jfly/worldcubeassociation.org/blob/6ad50b1c749d4140caca8f94d2781498dd7eea0a/WcaOnRails/.babelrc#L18) enabling this.